apps/portal/components/pagesComponents/_imagesScreen/pages/myVideos/ShareVideoModal.tsx

```
@@ -55,13 +57,15 @@ const ShareVideoModal: React.FC<IShareVideoModalProps> = ({
-            Copy Link
+            {t('modals.copyLink')}
           </AppButton>
         </div>
       }
     >
-      <h2 className="tw-mb-3 tw-text-2xl tw-font-semibold">Share link</h2>
-      <label>Link to video</label>
+      <h2 className="tw-mb-3 tw-text-2xl tw-font-semibold">
+        {t('modals.shareLink')}
+      </h2>
+      <label>{t('modals.linkToVideo')}</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)